tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -4657,6 +4690,61 @@ def _assert_is_fit(self, message_suffix: str = None):
                 error_message = f"{error_message} `.{message_suffix}`."
             raise AssertionError(error_message)
 
+    @Deprecated(min_version_to_warn="0.8", min_version_to_error="1.1", version_to_remove="1.1", new="persist")
+    def persist_models(self, **kwargs) -> List[str]:
+        """Deprecated method. Use `persist` instead."""
+        return self.persist(**kwargs)
+
+    @Deprecated(min_version_to_warn="0.8", min_version_to_error="1.1", version_to_remove="1.1", new="unpersist")
+    def unpersist_models(self, **kwargs) -> List[str]:
+        """Deprecated method. Use `unpersist` instead."""
+        return self.unpersist(**kwargs)
+
+    @Deprecated(min_version_to_warn="0.8", min_version_to_error="1.1", version_to_remove="1.1", new="model_names")
+    def get_model_names(self, **kwargs) -> List[str]:
```

Comment:
We will also need to update https://github.com/autogluon/autogluon/blob/master/docs/api.rst to ensure that all deprecated methods don't show up in the side bar on https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.html

Reply:
@gidler Do you know how we can hide certain methods from appearing in the side bar?

Reply:
Do you want the deprecated methods to not be documented in the methods table as well as the side bar, or is it just the side bar that we are concerned about?

I don't think this is as easy as one would wish it was. We are avoiding including `__init__` method documentation [here](https://github.com/autogluon/autogluon/blob/master/docs/_templates/autosummary/class.rst?plain=1#L16-L17) by filtering on the name. We could add the deprecated function names manually but that doesn't seem very maintainable and any function with that name across our whole project would get dropped.

Reply:
Ok, we can keep it for now, and if we can find a way to elegantly filter them, we can add it